src/games/strategy/triplea/ui/RouteOptimizer.java

```diff
@@ -145,70 +138,26 @@ public Point getLastEndPoint() {
   }
 
   /**
-   * Gives a List of Point arrays (Routes) which are the offset equivalent of the given points
-   * Size may vary depending on MapProperties
+   * Matrix Transpose method to transpose the 2dimensional point list
    * 
    * @param points A Point array
-   * @return Offset Point Arrays
+   * @return Offset Point Arrays including points
    */
-  private List<Point[]> getAlternativePoints(Point... points) {
-    List<Point[]> alternativePoints = new ArrayList<>();
-    if (isInfiniteX || isInfiniteY) {
-      int altArrayCount = getAlternativePointArrayCount();
-      for (int i = 0; i < altArrayCount; i++) {
-        alternativePoints.add(new Point[points.length]);
-      }
-      int counter = 0;
-      for (Point point : points) {
-        if (isInfiniteX) {
-          alternativePoints.get(0)[counter] = new Point(point.x - mapWidth, point.y);
-          alternativePoints.get(1)[counter] = new Point(point.x + mapWidth, point.y);
-        }
-        if (isInfiniteY) {
-          int index = altArrayCount == maxAdditionalScreens ? 2 : 0;
-          alternativePoints.get(index)[counter] = new Point(point.x, point.y - mapHeight);
-          alternativePoints.get(index + 1)[counter] = new Point(point.x, point.y + mapHeight);
-        }
-        if (isInfiniteX && isInfiniteY) {
-          alternativePoints.get(4)[counter] = new Point(point.x - mapWidth, point.y - mapHeight);
-          alternativePoints.get(5)[counter] = new Point(point.x - mapWidth, point.y + mapHeight);
-          alternativePoints.get(6)[counter] = new Point(point.x + mapWidth, point.y - mapHeight);
-          alternativePoints.get(7)[counter] = new Point(point.x + mapWidth, point.y + mapHeight);
+  public List<Point[]> getAllPoints(Point... points) {
```

Comment:
I don't think I feel strongly enough to ask for further test changes here. Multiple tests depending on one component indicates system coupling. That can get to some pretty bad places, so the rule of thumb is to try and have only one test failure for any one thing that is broken.